config/testgrids/kubernetes/sig-cloud-provider/azure/config.yaml

```diff
@@ -1,7 +1,9 @@
 dashboard_groups:
 - name: sig-azure
```

Comment:
```suggestion
- name: provider-azure
```

Reply:
thanks @justaugustus! I've updated the PR.